applications/leetcoin.md

```diff
@@ -107,81 +185,81 @@ Please also provide the GitHub accounts of all team members. If they contain no
-| 4. | Diversify Sets | Diversify problem sets to include security, testing etc.                                                                                                  |  
-| 5. | Educational Modules (Learner) | A dashboard where learners can explore different popular smart contracts to delve deeply into                                                             | 
-| 6. | Educational Modules (Educator) | A builder where educators can add rich text and video media for courses                                                                                   |  
-| 7. | Badges | Award badges to specific users and educators that have hit specific achievements                                                                          | 
+This milestone will extend the problem compiler and problem sets as well as introduce new functionality related to
+different learning paths and in-depth analysis of hugely popular smart contracts.
+
+| Number | Deliverable                    | Specification                                                                                                                                             |
+|-------:|--------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+|    0a. | License                        | Apache 2.0                                                                                                                                                |
+|    0b. | Documentation                  | We will provide the documentation for the repository code as well as a tutorial for publishing/designing a user-generated coding problem and test suites. |
+|    0c. | Testing Guide                  | Best practices for developing problem sets, how to test them and a FAQ.                                                                                   |
+|    0d. | Article                        | Announcement blog post explaining the new educational modules on the platform.                                                                            |
+|     1. | Set Creator                    | Implement problem set curation and creation                                                                                                               |  
+|     2. | Problem Favouriting            | Allow users to favourite specific problems to easily reference later                                                                                      |  
+|     3. | Expand Online Judge            | Allow for problems written in Ink!, Rust and Go by extending compiler                                                                                     |  
+|     4. | Diversify Sets                 | Define and add more problems.                                                                                                                             |  
```

Comment:
Is it okay to cut down on Educational modules for learners and educators? At this moment we are not sure if they will be beneficial since there are plenty of resources online.